test_runner/fixtures/zenith_fixtures.py

```diff
@@ -277,24 +263,48 @@ def generate_tenant_token(self, tenant_id):
         return token
 
 
+@zenfixture
+def worker_seq_no(worker_id: str):
```

Comment:
It took me a while to understand this, until I realized that 'worker_id' is a fixture that  comes from `pytest-xdist. Some comments would be nice.